sphinx_immaterial/custom_admonitions.py

```diff
@@ -257,7 +281,23 @@ def on_builder_inited(app: Sphinx):
     custom_admonitions: List[CustomAdmonitionConfig] = getattr(
         config, "sphinx_immaterial_custom_admonitions"
     )
+    custom_admonition_names = []
     for admonition in custom_admonitions:
+        custom_admonition_names.append(admonition.name)
+        if admonition.name in VERSION_DIR_STYLE:  # if specific to version directives
+            if admonition.icon is None:  # for default icon
```

Comment:
could simplify this to:

`admonition.icon = load_svg_into_builder_env(app.builder, admonition.icon or VERSION_DIR_STYLE[admonition.name]["icon"])`

Reply:
I had a feeling you were going to request that (after I pushed it).